ports/mimxrt10xx/Makefile

```diff
@@ -79,23 +79,24 @@ ifeq ($(OS),Windows_NT)
 	SDPHOST = sdphost/win/sdphost.exe
 else
 	UNAME_S := $(shell uname -s)
-	UNAME_I := $(shell uname -i)
+	UNAME_M := $(shell uname -m)
 
 	ifeq ($(UNAME_S),Darwin)
 		SDPHOST = sdphost/mac/sdphost
 	endif
 
 	ifeq ($(UNAME_S),Linux)
-		ifeq ($(UNAME_I),x86_64)
+		ifeq ($(UNAME_M),x86_64)
 			SDPHOST = sdphost/linux/amd64/sdphost
-	  endif
-
-	  ifeq ($(UNAME_I),i386)
+		endif
+		ifeq ($(UNAME_M),i386)
 			SDPHOST = sdphost/linux/i386/sdphost
-	  endif
+		endif
 	endif
 endif
 
+SDPHOST ?= false

```

Comment:
maybe we should have a better error message with

```
ifndef SDPHOST
$(error SDPHOST is not found for this machine !!)
endif
```

Reply:
Thanks, I improved this.  Now, if it's not detected,
```
$ make flash-sdp BOARD=imxrt1010_evk
SDPHOST is not found for this machine
```
however, other targets can still be built even if this happens.

Reply:
that is even better